services/clusters-keeper/src/simcore_service_clusters_keeper/data/docker-compose.yml

```diff
@@ -123,16 +144,51 @@ services:
       retries: 50
     volumes:
       - redis-data:/data
+    networks:
+      - cluster
     deploy:
       placement:
         constraints:
           - "node.role==manager"
+      resources:
+        limits:
+          memory: 512M
+          cpus: "0.5"
+
+  prometheus:
+    image: prom/prometheus:v2.51.0
+    ports:
+      - 9090:9090
+    configs:
+      - source: prometheus-config
+        target: /etc/prometheus/prometheus.yml
+    volumes:
+      - prometheus-data:/prometheus
+      - /var/run/docker.sock:/var/run/docker.sock:ro
+    user: root # because of docker
+    networks:
```

Comment:
I am missing at least:
```
- "--storage.tsdb.retention=${JEEPERS_KREEPERS_PROMETHEUS_RETENTION}"
```

Reply:
added retention.size=1GB for now.. the AMI is a bit limited in size